cmd/npmExecuteLint.go

```diff
@@ -186,3 +207,11 @@ func findEslintConfigs(utils lintUtils) []string {
 	}
 	return eslintConfigs
 }
+
+func prepareArgs(defaultArgs []string, outputFileNamePattern, outputFileName string) []string {
+	if outputFileName != "-" { // in this case we omit the -o flag and output will go to the log
```

Comment:
I'm open for other approaches, 

e.g.
- `vi -` reads from `stdin`, like we write to `stdout`
- 'curl --output -` writes to `stdout`.

Seems more the less a common practice

Reply:
providing `/dev/stdout` as file name should also work, but well ... haha

Reply:
I too would opt for checking for empty string for outputFileName instead of "-".

Reply:
I adjusted to to empty string instead of dash.

On the command-line we have now to say: ''./piper npmExecuteLint --failOnError --outputFormat checkstyle --outputFileName ""` (empty double quotes), which I find less intuitiv compared to having a dash ...